orchestration/scripts/validate-checksums.py

```diff
@@ -0,0 +1,50 @@
+import google.auth
+from google.cloud import bigquery
+import json
+import os
+
+# Take contextual information as arguments.
+project_id = os.environ["PROJECT_ID"]
+dataset_name = os.environ["DATASET_NAME"]
+load_tag = os.environ["LOAD_TAG"]
+
+# Set up BigQuery client.
+credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+bqclient = bigquery.Client(credentials=credentials, project=project_id,)
+
+
+# Log a checksum error for the given source and target path.
+def log_checksum_error(source_name: str, target_path: str):
+    error_log = {
+        "errorType": "ChecksumError",
+        "filePath": source_name,
+        "fileName": source_name.split("/")[-1],
+        "message": f"New crc32c checksum does not match the original for the file ingested to {target_path}."
+    }
+    with open('../../logs/errors.log', 'a+') as log_file:
+        log_file.write(json.dumps(error_log) + "\n")
```

Comment:
I had been debating whether I should do this step of writing to the file at the end of the script. I think it might make the code a bit more complicated but would avoid opening and writing to the file multiple times if a bunch of errors are thrown (which hopefully shouldn't be the case). Let me know if anyone has a strong opinion.